src/ripple/rpc/handlers/Tx.cpp

```diff
@@ -130,7 +130,7 @@ Json::Value doTx (RPC::Context& context)
                     lgr->seq (), *rawMeta, context.app.journal ("TxMeta"));
                 okay = true;
                 auto meta = txMeta->getJson (0);
-                addPaymentDeliveredAmount (meta, context, txn, txMeta);
+                addPaymentDeliveredAmount (meta, context, txn, *txMeta);
                 ret[jss::meta] = meta;
```

Comment:
move meta?

Reply:
Fixed in: `3464b7117 * delivered_amt [fold] Resolve feedback from Mickey 1`